modules/web/src/app/dynamic/enterprise/cluster-backups/list/backups/add-dialog/component.ts

```diff
@@ -105,6 +105,9 @@ export class AddClustersBackupsDialogComponent implements OnInit, OnDestroy {
     this.form.get(Controls.Destination).disable();
     this.getClusterNamespaces(this.projectID, this.cluster.id);
 
+    // this feild for now will be disable untill we have a list of BSL
+    this.form.get(Controls.Destination).disable();
```

Comment:
This field is already disabled above (Line# 105). Also, do we not have list of BSL now?